### DISCUSSION TITLE
How much pruning can I do as a validator node?

### DISCUSSION THREAD

**Discussion by jimmychu0807** [erigontech/erigon#7423] [Q&A] [answered]
Hi I am trying to setup a validator node in Eth testnet, while I also have some concern on the storage size. Is there an optimal balance between pruning certain states while still being able to run as an  "Ethereum full node"? Reading Erigon docs, it seems it is either an archive node, or a light client.

I found this [reddit thread](https://www.reddit.com/r/ethstaker/comments/100ngfy/whats_the_recommended_erigon_pruning_setting_for/), is:

```
--prune=htc --prune.r.before=11184524
```

kind of the golden setting? Thank you.

**Comment by AskAlexSharov** [comment, marked as answer]
As I remember --prune.r.before=11184524 is default value of this flag. so, I think it's equal to `--prune=hrtc`
we know that this value is big (db is not small), but we didn't work on storing less receipts for ConsensusLayer yet.
